### DISCUSSION TITLE
Donut - show percent values

### DISCUSSION THREAD

**Discussion by sadden3194** [apexcharts/apexcharts.js#2890] [Q&A]
Hi ! 

is it possible to show the percent values of each element in the middle of the donut instead the value?

<img width="863" alt="Bildschirmfoto 2022-01-13 um 20 26 33" src="https://user-images.githubusercontent.com/17011498/149396375-554a4ac8-ba3e-4afc-abce-87cf1cc37f95.png">

**Comment by brianlagunas** [comment]
Try using the formatter

https://apexcharts.com/docs/options/plotoptions/pie/

**Comment by sadden3194** [comment]
i tried this:
```
plotOptions: {
				pie: {
					donut: {
						labels: {
							show: true,
							value: {
								formatter: function (val) {
									return val;
								}
							}
						}
					}
				}
			},
```

but this will show the value, not the "percent value"

**Reply by brianlagunas** [reply]
you have to modify the formatter to return what you want

**Comment by sadden3194** [comment]
and how can I access the percent value??
